src/SourceBuild/content/eng/unifiedBuildValidation.targets

```diff
@@ -0,0 +1,19 @@
+<Project>
+
+  <PropertyGroup>
+    <UnifiedBuildValidationTestsDir>$([MSBuild]::NormalizeDirectory('$(RepoRoot)', 'test', 'Microsoft.DotNet.UnifiedBuild.Tests'))</UnifiedBuildValidationTestsDir>
+    <UnifiedBuildValidationTestsProject>$([MSBuild]::NormalizePath('$(UnifiedBuildValidationTestsDir)', 'Microsoft.DotNet.UnifiedBuild.Tests.csproj'))</UnifiedBuildValidationTestsProject>
+  </PropertyGroup>
+
+  <Target Name="RunUnifiedBuildValidation"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion">
+
+    <MSBuild Projects="$(UnifiedBuildValidationTestsProject)"
+             Targets="VSTest"
+             Properties="VsTestUseMSBuildOutput=true;
+                         VSTestLogger=trx;" />
```

Comment:
I think you can remove this line if you set `<UseVSTestRunner>true</UseVSTestRunner>` in Versions.props after L5: https://github.com/dotnet/installer/blob/58e4c4f2e4fc66597ebf4ff54809dc88f286e217/src/SourceBuild/content/eng/Versions.props#L5

This makes testing in this repo use the VSTest runner by default.